metrics/web/test/debug_menu/domain/entities/local_config_test.dart

```diff
@@ -0,0 +1,20 @@
+import 'package:flutter_test/flutter_test.dart';
+import 'package:metrics/debug_menu/domain/entities/local_config.dart';
+
+// ignore_for_file: prefer_const_constructors
+
+void main() {
+  group("LocalConfig", () {
+    test(
+      "creates an instance with the given is fps monitor enabled",
```

Comment:
```suggestion
      "creates an instance with the given parameters",
```
Or if we want to write similar tests for each field: 
```suggestion
      "creates an instance with the given is fps monitor enabled parameter",
```